content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -1,67 +1,236 @@
-your ticket page, click on *Refund* and select the corresponding *Invoice*. Clicking on *Reverse*
-generates a credit note, and you can *Post* it while still being in the *Helpdesk* app.
+By default, the quantity will match the validated quantity from the delivery order. Update the
+:guilabel:`Quantity` field if necessary.
 
-.. image:: after_sales/aftersales2.png
+.. image:: after_sales/after-sales-reverse-transfer.png
    :align: center
+   :alt: View of a reverse transfer creation page.
 
-Allow product returns from tickets
-----------------------------------
+Click :guilabel:`Return` to confirm the return. This generates a new warehouse operation for the
+incoming returned product(s). A :guilabel:`Returns` smart button will then be added to the top of
+the :menuselection:`Helpdesk` :guilabel:`Ticket`.
 
-The process of a product return from your customer back to your warehouse is taken into action when,
-at the ticket page, you choose the option *Return*.
+.. image:: after_sales/after-sales-return-smart-button.png
+   :align: center
+   :alt: View of the return smart button on a helpdesk ticket.
+
+.. seealso::
+   :doc:`/applications/sales/sales/products_prices/returns`
+
+Send products for repair from a ticket
+======================================
 
-.. image:: after_sales/aftersales3.png
+If the :guilabel:`Ticket` is related to an issue with a faulty or broken product, a repair order
+can be created from the :menuselection:`Helpdesk`, and managed through the :menuselection:`Repair`
+application.
+
+To create a new repair order, click :guilabel:`Repair`.
+
+.. image:: after_sales/after-sales-repair-reference.png
    :align: center
+   :alt: View of a repair reference page.
+
+If a product was specified in the :guilabel:`Product` field on the ticket, it will be automatically
+added to the :guilabel:`Product to Repair` field on the newly created :guilabel:`Repair Reference`.
+If applicable, add the :guilabel:`Sale Order` and :guilabel:`Return Order` from which the faulty
+product originates.
 
-Grant coupons from tickets
---------------------------
+Select the :guilabel:`Invoice Method`. If the customer will not be charged for this repair, this
+field can be left unchanged.
 
-First, be sure to have your *Coupon Program* planned in the *Sales* or *Website* application. Then,
-in *Helpdesk*, open your ticket, click on *Coupon*, and choose the respective one.
+If parts are required for the repair, they can be added in the :guilabel:`Parts` tab. Services can
+be added as product lines on the :guilabel:`Operations` tab. Additional information for the internal
+repair team can be added to the :guilabel:`Repair Notes` tab. Information for the customer can
+be added to the :guilabel:`Quotation Notes` tab, and will be automatically added to the PDF of the
+quotations generated from this :guilabel:`Repair Reference`.
 
-.. image:: after_sales/aftersales4.png
+.. image:: after_sales/after-sales-repair-tabs.png
```

Comment:
might not need this since users can see/read the tabs right there on the form (it's obvious)
the line item there doesn't communicate much I think?